arcgis-ios-sdk-samples/Maps/Mobile map (search and route)/README.md

```diff
@@ -33,6 +33,10 @@ A list of maps from a mobile map package will be displayed. If the map contains
 * AGSRouteTask
 * AGSTransportationNetworkDataset
 
+## Offline data
+
+This sample uses the [San Francisco](https://www.arcgis.com/home/item.html?id=260eb6535c824209964cf281766ebe43) mobile map package and the [Yellowstone](https://www.arcgis.com/home/item.html?id=e1f3a7254cb845b09450f54937c16061) mobile map package. Both are downloaded from ArcGIS Online automatically.  
```

Comment:
I don't see either one of these in `PortalItems.plist`. Am I missing something?

Reply:
Vivian can elaborate more here. 

These portal items predate the on-demand "sideload" workflow, and are stored in the `Shared resources` directory in the project, rather than `Portal Data`. Viv and I agreed to leave it to your call on either redo them with the new sideload mechanism, or just replace the existing data with the new ones.

Reply:
You might be confused because I asked you (Phil) about moving data from `Shared resources` to `Portal data`. I was actually asking about a different issue about tile packages. I finished replacing these existing files prior to our conversation so I didn't apply your suggestion. Let me revert this PR to a draft and move these files to `Portal data`. Sorry about the confusion!